content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/serial_numbers.rst

```diff
@@ -1,147 +1,230 @@
-you received and copy the list. Then, past them in the *Lot/Serial
-Number Name* column of the wizard. Doing so, Odoo will automatically
-create the lines you need.
-
-.. image:: serial_numbers/serial_numbers_08.png
-    :align: center
-
-.. image:: serial_numbers/serial_numbers_09.png
-    :align: center
-
-.. image:: serial_numbers/serial_numbers_10.png
-    :align: center
-
-Operation types
-===============
-
-You also have the possibility to define how you will manage lots for
-each operation type. To define it, go to :menuselection:`Configuration --> Operation Types` in the
-*Inventory* app.
-
-For each type, you can decide if you allow the creation of new lot
-numbers or want to use existing ones. By default, the creation of new
-lots is only allowed at product reception.
-
-.. image:: serial_numbers/serial_numbers_11.png
-    :align: center
+New serial numbers can be created for products already in stock with no assigned serial number. To
+do this, go to :menuselection:`Inventory --> Products --> Lots/Serial Numbers`, and click
+:guilabel:`Create`. A new :guilabel:`Lot/Serial Number` will be generated automatically.
 
 .. tip::
-      If you have inter-warehouse transfers and track products by lots, it can
-      be useful to allow using existing lot numbers in receipts too.
+    Although Odoo automatically generates a new :guilabel:`Lot/Serial Number` to follow the most
+    recent number, it can be edited and changed to be whatever number is desired by clicking the
+    line under the :guilabel:`Lot/Serial Number` field and changing the generated number.
+
+Once the :guilabel:`Lot/Serial Number` is generated, click the :guilabel:`drop-down` next to
+:guilabel:`Product`, and select the product that this new number will be assigned to.
+:guilabel:`Save` changes.
+
+.. image:: serial_numbers/serial-numbers-new-serial-number.png
+   :align: center
+   :alt: New serial number created for existing product stock.
+
+After the new serial number has been created and assigned to the desired product, navigate back to
+the product form by going to :menuselection:`Products --> Products`, and selecting the product the
+new serial number was just assigned to. Click the :guilabel:`Lot/Serial Numbers` smart button to
+see the new serial number.
+
+Manage serial numbers for shipping and receiving
+================================================
+
+Manage serial numbers on receipts
+---------------------------------
+
+Assigning serial numbers to *incoming* goods can be done directly from the purchase order. To
+create a purchase order, go to the :guilabel:`Purchase` app, and click :guilabel:`Create` to create
+a new :abbr:`RFQ (Request for Quotation)`. Then, fill out the information by adding a
+:guilabel:`Vendor`, and by adding products to the :guilabel:`Product` lines by clicking
+:guilabel:`Add a product`. Choose the desired quantity to order by changing the number in the
+:guilabel:`Quantity` column.
```

Comment:
```suggestion
:guilabel:`Quantity` column. Click :guilabel:`Confirm Order`. This will convert the :abbr:`RFQ (Request for Quotation)` to a :guilabel:`Purchase Order`.
```